backend/cmd/migrate/migrations/000008_topology_cache_add_composite_key.down.sql

```diff
@@ -0,0 +1,2 @@
+ALTER TABLE topology_cache DROP CONSTRAINT topology_cache_pkey;
+ALTER TABLE topology_cache ADD CONSTRAINT topology_cache_id_key UNIQUE (id);
```

Comment:
and i think this should be
```suggestion
ALTER TABLE topology_cache ADD PRIMARY_KEY (id);
```
primary keys are inherently unique but it seems this table would no longer have a PK after this ran

Reply:
`id` is not a PK its just a UNIQUE key